### DISCUSSION TITLE
Issue for deprecation warning?

### DISCUSSION THREAD

**Discussion by pohjolal** [laravel/framework#45410] [General] [answered]
Should I create a new issue about deprecation warning? As it is not a bug as such I am not sure. 
lararavel/socialite generates deprecation warnings (null as string value) in SocialiteManager.php formatRedirectUrl() on php 8.1 if config value redirect is not set.

**Comment by henzeb** [comment, marked as answer]
I'd say do so... as it should be fixed at some point anyway.
